src/components/structures/login/ForgotPassword.js

```diff
@@ -150,7 +148,8 @@ module.exports = React.createClass({
         var ErrorDialog = sdk.getComponent("dialogs.ErrorDialog");
         Modal.createDialog(ErrorDialog, {
             title: title,
-            description: body
+            description: body,
+            button: _t("OK"),
```

Comment:
The problem is that OK is not an universal word, and strings in getDefaultProps won't get translated, since they are set before language is loaded. So instead of stopping everything to load to wait languages to load, we decided to suggest developers that getDefaultProps don't have translateable strings.